test/integration/targets/vmware_vm_facts/tasks/main.yml

```diff
@@ -1,5 +1,5 @@
 # Test code for the vmware_vm_facts module
-# Copyright: (c) 2017, Abhijeet Kasurde <akasurde@redhat.com>
+# Copyright: (c) 2017-2018, Abhijeet Kasurde <akasurde@redhat.com>
```

Comment:
Based on internal discussions, please leave the copyright date as 2017.  See https://github.com/ansible/ansible/issues/45989 for details.